csharp/build_packages.bat

```diff
@@ -1,7 +1,7 @@
 @rem Builds Google.Protobuf NuGet packages
 
-dotnet restore src
-dotnet pack -c Release src\Google.Protobuf || goto :error
+dotnet restore src/Google.Protobuf.sln
+dotnet pack -c Release src/Google.Protobuf.sln || goto :error
```

Comment:
Ooh, I meant to put `<IncludeSymbols>true</IncludeSymbols>` in the project file. (I'd prefer to have everything around the packaging in one place.)

I don't think we need to include the source in the main package itself though.

Will fix the project file now...